.github/workflows/uBench_whole_chain.yml

```
@@ -0,0 +1,90 @@
+name: Test the entire build and invokation chain of test examples.
```

the name of the workflow is too long. 

please rename the file to `function-composition.yml`. We are going to add other functions later

```suggestion
name: Function composition tests
```